src/main/editDelete/injectTakeInfo.ts

```diff
@@ -1,11 +1,11 @@
-import { IndexRange, Word } from 'sharedTypes';
+import { IndexRange, TakeGroup, Word } from 'sharedTypes';
```

Comment:
Same issue here

Reply:
Type imports are fine to do as absolute I think